bodhi/server/services/updates.py

```diff
@@ -364,7 +372,9 @@ def query_updates(request):
 
     user = data.get('user')
     if user is not None:
-        query = query.filter(Update.user == user)
+        query = query.filter(
+            Update.user == user
+        )
```

Comment:
This change isn't related to the purpose of the patch, and I also happen to prefer the original styling.